src/utils/errorDialog.js

```diff
@@ -0,0 +1,36 @@
+import {
+  Alert,
+} from 'react-native';
+import { Component } from 'react';
+import store from 'datalayer/store';
+import { logoutDispatch } from 'datalayer/actions/auth.action';
+
+class ErrorDialog extends Component {
```

Comment:
Why using React Component here?

Reply:
At first I wanna use the logoutDispatch() using props but then I decided to use store.dispatch() to make it easier.
Forgot to refractor this one